cypress/integration/context-selection/data-set.feature

```diff
@@ -0,0 +1,49 @@
+Feature: A data set can be selected
+
+    Scenario: No value is being displayed
+        Given no data set has been selected yet
+        Then a no-value-label should be displayed
+
+    Scenario: A value is being displayed
+        Given some selectable data set exists
+        And a data set has been selected
+        Then a value-label should be displayed
+
+    Scenario: The selectable data sets are being loaded
+        Given the selectable data sets are being loaded
+        When the user opens the menu
+        Then nothing but a loading message should be displayed
+
+    #
+    # @TODO: Stubbing the request with a 404 response doesn't work
+    # unless we provide "retry: false" to the react-query client options
+    #
+    ## Scenario: The selectable data sets could not be loaded
+    ##     Given loading the selectable data sets failed
+    ##     When the user opens the menu
+    ##     Then nothing but an error message should be displayed
```

Comment:
@ismay do you have an idea what I can do to test error states?
react-query retries the query and just shows the loading indicator.
When I set `retry: false` in the react-query client's options, this test works

Reply:
Ah yeah, there's a couple recommendations for testing, turning off retries is one of them: https://react-query.tanstack.com/guides/testing#turn-off-retries

Reply:
Can be done globally with a provider, or per hook

Reply:
We've discussed this on slack and I've turned it off for cypress